### ISSUE TITLE
unable to remove windows

### ISSUE THREAD

**Issue by jessereynolds** [dylanratcliffe/onceover#41]
because my control repo doesn't support windows, and throws a fail() during compilation for windows, i'm trying to coerce onceover to not try running specs with windows factsets. 

I'm currently getting this error:

```
Error while running: #<RuntimeError: Could not find  in list of classes, nodes or groups>
```

with the following onceover.yaml:

```yaml
classes:
  - role::test

nodes:
  - CentOS-6.6-32
  - CentOS-6.6-64

class_groups:

node_groups:
  non_windows_nodes:
    include: 'all_nodes'

test_matrix:
  - all_nodes:
      classes: 'all_classes'
      tests: 'spec'
```

**Comment by jessereynolds** [comment]
Duh, if I just remove the lines nested under node_groups: it works fine. The error message received could be improved though :-)

**Comment by dylanratcliffe** [comment]
Fixed in 3.0.3
